crates/editor/src/editor.rs

```diff
@@ -4110,6 +4110,39 @@ impl Editor {
                 active_index: 0,
                 ranges: tabstops,
             });
+
```

Comment:
There's two things that might go wrong with this:

1. A snippet might have more than 1 tab-stop with auto-closable brackets
2. Each tab-stop might have more selections

Think of a snippet like this:


```
function <myFunctionName>(<arg1>, <arg2>) {
  console.log("this is ", <myFunctionName>);
  const myString = <myFunctionName>;
}
```

So in this case the snippet has 3 tab stops: `myFunctionName,` `arg1, `arg2`, and one of them -- `<myFunctionname>` -- has 3 selections.

It's probably best to check all selections for each tab stop.